package.json

```diff
@@ -1,29 +1,30 @@
 {
   "name": "nextjs-project",
   "version": "1.0.0",
+  "private": "true",
   "description": "Next.js starter kit for Wunder projects.",
   "main": "index.js",
   "directories": {
     "doc": "docs"
   },
   "dependencies": {
-    "next": "^12.1.6",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "next": "^13.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@types/node": "^16.11.7",
-    "@types/react": "^17.0.43",
-    "@typescript-eslint/eslint-plugin": "^5.18.0",
-    "@typescript-eslint/parser": "^5.18.0",
-    "eslint": "^8.16.0",
+    "@types/node": "^18.11.7",
```

Comment:
If the node version of the project is 16 I think it makes sense to leave the node-types version at 16 too?

Reply:
Agreed - fixed in https://github.com/wunderio/nextjs-project/pull/217/commits/1fe03fd1198de4e9deb8798bbcd9d8d6a17554a1